rudder-web/src/main/scala/bootstrap/liftweb/LiftInitContextListener.scala

```diff
@@ -105,32 +103,4 @@ class LiftInitContextListener extends ContextLoaderListener {
     super.contextDestroyed(sce)
   }
 
-  private[this] def initSpringAuthentication(): Unit = {
-    import RudderProperties.{config => c}
-
-    val springConfigFile = if( c.getBoolean("rudder.auth.ldap.enable")) {
```

Comment:
That property should be removed from configuration file but myabe it is already removed ;)